cfme/configure/configuration/diagnostics_settings.py

```diff
@@ -141,7 +141,7 @@ class CollectLogsBase(Pretty, NavigatableMixin, Updateable):
             uri: depot uri
             username: depot username
             password: depot password
-            second_server_collect: Set True to use slave server
+            second_server_collect: Set True to use second server
```

Comment:
Q: why the change in name? Just for increased readability?

Reply:
I guess `master` is not really a correct term because the server you login to is your master/current. That's what I think Tasos wanted to change it for.

Reply:
Yes, instead of master/slave, these parameters/methods/classes are actually describing current vs. non-current UI server. I've settled on primary / second[ary] as the terminology to use.